atlasdb-client/src/main/java/com/palantir/atlasdb/util/AtlasDbMetrics.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright 2017 Palantir Technologies
- *
+ * <p>
```

Comment:
Yeah, this is a side effect of the IntelliJ auto-formatter because AtlasDB checkstyle setup is incorrect (see #1693 )